vllm/model_executor/models/minicpmv.py

```diff
@@ -69,6 +70,15 @@
 }
 
 
+class MiniCPMVImageInput(TypedDict):
+    """Image input to MiniCPM-V model with image bound auxiliary data."""
+
+    image: Image.Image
+
+    # If provided, contains image_bounds data for the whole prompt.
+    image_bounds: Optional[torch.Tensor]
```

Comment:
```suggestion
    image_bounds: NotRequired[torch.Tensor]
```

Should use `NotRequired` from `typing_extensions`.